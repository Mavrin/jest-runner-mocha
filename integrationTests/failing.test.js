const runJest = require('./runJest');

const normalize = res => res.replace(' [ERR_ASSERTION]', '');

it('Works when it has failing tests', () => {
  return expect(runJest('failing').then(normalize)).resolves.toMatchSnapshot();
});
