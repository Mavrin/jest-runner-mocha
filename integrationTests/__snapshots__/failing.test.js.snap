// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works when it has failing tests 1`] = `
"FAIL integrationTests/__fixtures__/failing/__mocha__/__tests__/file.test.js
    My tests This test fails
      [31mAssertionError: 1 == 2[0m
      [32m+ expected[0m [31m- actual[0m
      [31m-1[0m
      [32m+2[0m
      at mocked-stack-trace
,
    My tests This test also fails
      [31mAssertionError [ERR_ASSERTION]: 3 == 4[0m
      [32m+ expected[0m [31m- actual[0m
      [31m-3[0m
      [32m+4[0m
mocked-stack-trace
  âœ• This test fails
  âœ• This test also fails
Test Suites: 1 failed, 1 total
Tests:       2 failed, 2 total
Snapshots:   0 total
Time:
Ran all test suites.

"
`;
